homeassistant/components/modbus/__init__.py

```diff
@@ -257,7 +257,6 @@ class ModbusHub:
 
     def __init__(self, client_config):
         """Initialize the Modbus hub."""
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.